src/lib/reminderFields.ts

```diff
@@ -29,16 +28,6 @@ export const buildReminderFields = (today: Date = new Date()): ReminderFields =>
     };
 };
 
-export const getReminderFields = (
-    variant: EpicVariant,
-    targeting: EpicTargeting,
-): ReminderFields | undefined => {
-    const userCohorts = getUserCohorts(targeting);
-    const isSupporter = userCohorts.includes('AllExistingSupporters');
-
-    if (isSupporter) {
```

Comment:
this logic isn't needed - we can enabled/disable the reminder cta from the tool now